internal/web/test/admin_test.go

```diff
@@ -212,13 +212,10 @@ func TestAdminInvitation(t *testing.T) {
 	require.NoError(t, err)
 	invitation1, err := db.GetInvitationByID(1)
 	require.NoError(t, err)
-	require.Equal(t, invitation1, &db.Invitation{
-		ID:        1,
-		Code:      invitation1.Code,
```

Comment:
I left this one out because it would be a tautological assertion.

I also discovered [cmpopts](https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#EquateApproxTime), but it didn't seem worth an extra dependency to me.